qt-models/diveplannermodel.cpp

```diff
@@ -894,8 +894,8 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 				add_dive_to_trip(copy, current_dive->divetrip);
 			record_dive(copy);
 			QString oldnotes(current_dive->notes);
```

Comment:
This mechanism to keep user entered notes above the planner output is only activated in mode "save replanned dive as new". But it should also work in mode "save replanned dive (overwrite)".